src/library/components/navigation/navPrimary/NavPrimary.scss

```diff
@@ -1,23 +1,11 @@
-/* stylelint-disable no-descending-specificity */
 @import '../../../../styles/abstracts/abstracts';
 @import '../../../../styles/base/base';
 @import '../../../../styles/utilities/utilities';
 
-// These are one of sizes to get the icons to match the font sizes
-$icon-size-sm: space(lg);
-$icon-size: 1em;
-$icon-size-lg: space(2x);
-$panels-width: 100vw;
-$num-panels: 4;
-$nav-bar-z-index: 700;
-$nav-back-z-index: 600;
-$nav-menu-z-index: 550;
-$nav-list-z-index: 500;
-$nav-search-z-index: 550;
-
-// Used in nav-menu nav-links
-$nav-link-padding: space(2x) space(3x);
-$menu-padding: space(md) 0;
+@import './VarsNavPrimary';
```

Comment:
thoughts on file naming conventions? we have a mix of kebab and pascal

Reply:
At previous stops, css files that match a component are named after the component, so `ComponentName.tsx` would have a `ComponentName.scss` counterpart. Other files would either be partials, so they'd be `_partialName.scss`, or mixins, so `MixinName.scss`.